README.md

```diff
@@ -886,6 +887,31 @@ module.exports = {
 };
 ```
 
+### `exportGlobals`
+
+Type: `Boolean`
+Default: `false`
+
+Allow `css-loader` to export names from global class or id, so you can use that as local name.
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        loader: 'css-loader',
+        options: {
+          exportGlobals: true,
```

Comment:
put that option to the `modules` option:
```
modules: { exportGlobals: true  }
```

Reply:
Ok, thx.